docs/en/siem/machine-learning.asciidoc

```diff
@@ -2,12 +2,12 @@
 [role="xpack"]
 == Anomaly Detection with Machine Learning
 
-For *https://www.elastic.co/cloud/elasticsearch-service/signup[Free Trial]*
+For *Free Trial*, *https://www.elastic.co/cloud/elasticsearch-service/signup[Cloud]*
```

Comment:
Isn't it only available on Cloud w/ Platinum license (or free trial)? I'm unclear on the distinction between Free Trial and Cloud here. Or is this just saying that the service itself is available on cloud -- in terms of distribution as opposed to licensing (free trial/platinum)?

https://www.elastic.co/products/elasticsearch/service/pricing

Reply:
You can access the platinum features when you use the free trial on local installations. For cloud, everything is enabled by default and free during the free trial period.

Just wanted users to know they can try everything out for free locally as well.